docs/legacy/guide/opentelemetry-elastic.asciidoc

```diff
@@ -390,6 +392,48 @@ Here is an example of AWS Lambda Node.js function managed with Terraform and the
 
 * https://github.com/michaelhyatt/terraform-aws-nodejs-api-worker-otel/tree/v0.23[Sample Terraform code]
 
+[float]
+[[elastic-open-telemetry-resource-attributes]]
+==== Resource attributes
+
+A resource attribute is a key/value pair containing information about the entity producing telemetry.
+Resource attributes are mapped to Elastic Common Schema (ECS) fields like `service.*`, `cloud.*`, `process.*`, etc.
+These fields describe the service and the environment that the service runs in.
+
+The examples below set the Elastic (ECS) `service.environment` field for the resource, i.e. service, that is producing trace events.
```

Comment:
Should we explicitly call out that `deployment.environment` (OpenTelemetry attribute) is mapped to `service.environment` (ECS)?

Reply:
Yup. Good idea 👍.